Libraries/Image/Image.android.js

```diff
@@ -79,6 +79,7 @@ var ImageSpecificStyleKeys = new Set(Object.keys(ImageStylePropTypes).filter(x =
 var Image = React.createClass({
   propTypes: {
     ...ViewPropTypes,
+    children: PropTypes.element,
```

Comment:
`<Image>` with children is deprecated and this feature will be removed very soon. See `<ImageBackground>` for more details.